lib/Graph/Nodes.cpp

```diff
@@ -519,12 +519,14 @@ bool ChannelwiseQuantizedConvolutionNode::verify() const {
                                getScales().dims().size(), size_t(1), this);
 
   // check qparam sizes
+  /*
```

Comment:
Because this checks if group size equals to channel size, which is not guanranteed in channelwise conv.